vnext/local-cli/templates/cpp/proj/MyApp.vcxproj

```diff
@@ -102,6 +102,7 @@
       <PrecompiledHeader>Use</PrecompiledHeader>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
+      <ForcedIncludeFiles>pch.h</ForcedIncludeFiles>
```

Comment:
We should try to avoid this for template projects, where we want to give end-users more control. I remember Mso required this. Does this change add usage of it outside of the boundary?

Reply:
I am adding dependency on jsi.cpp. It fails to compile because projects wants to see the pch.h as the first include. I have rolled bacj this change and instead set the jsi.cpp explicitly to not use pch.

---
In reply to: [460425452](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r460425452) [](ancestors = 460425452)